prime-router/src/main/kotlin/Element.kt

```diff
@@ -244,7 +277,101 @@ data class Element(
     }
 
     /**
-     * Take a formatted CsvField value and turn into a normalized value stored in an element
+     * Take a formatted value and check to see if can be stored in a report.
+     */
+    fun checkForError(formattedValue: String, format: String? = null): String? {
+        if (formattedValue.isBlank() && !isOptional && canBeBlank != true) return "Blank value for element '$name'"
```

Comment:
New function that returns error details and guards against formatting exceptions.